examples/chart/teleport-cluster/.lint/disable-local-authentication.yaml

```diff
@@ -0,0 +1,3 @@
+clusterName: helm-lint
+authenticationType: github
+disableLocalAuthentication: true
```

Comment:
I'm inclined to suggest that we change this to a nested YAML structure like:
```yaml
authentication:
  type: github
  local_auth: false
```

That way it nests all authentication-related settings under one header and makes things easier to follow, and will be easier to extend in future if needed.

It'd need some logic to coalesce `authenticationType` when set for backwards compatibility.

Reply:
Done. I left `authenticationSecondFactor` out of the `authentication` object as this implies a bigger refacto. I'm not sure if current state is OK, if I should continue the refacto here or move to a new issue. WDYT?

Reply:
As we're introducing a new `authentication` top level object here, I'd lean towards also refactoring the `authenticationSecondFactor` item out (and adding a `coalesce` for backwards compatibility) at the same time. It's ultimately up to @r0mant though!